src/concatArray.js

```diff
@@ -0,0 +1,6 @@
+import { reduce } from 'ramda';
+import { concat } from 'most';
+
+export default reduce((main$, child$) => {
```

Comment:
I'd follow the convention of the rest of the library of having a static function not exported which is named. For example:

```
const concatArray = ...

export default concatArray;
```